packages/ant-util-analytics/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@back4app/ant-util-analytics",
+  "version": "0.0.17",
+  "publishConfig": {
+    "access": "public"
+  },
+  "engines": {
+    "node": ">=10.10"
+  },
+  "preferGlobal": true,
```

Comment:
This line should be removed. It means that you want it to be installed with npm install ant-util-analytics --global and I think that's not the case, right?

Reply:
Yes, makes no sense to install It globally :)